### DISCUSSION TITLE
Custom actions on the toast bar?

### DISCUSSION THREAD

**Discussion by goforbg** [fkhadra/react-toastify#626] [Ideas]
First of all, I'd like to say I'm a huge fan of the project and have been using it since I learned to code in React JS. 

### 💡 I was wondering if we could include **actions** in the toast. 
 
A toast helps in saying _something happened_.
Actions could extend the value for the user, by saying _something happened, here's what you can do about it._

**Examples**

1. Toast: Your session has been expired.
    Toast + Actions: Your session has expired, click on this toast to log in again. 

I guess I'm talking about a very similar implementation of [snack bars by Material UI](https://material-ui.com/components/snackbars/)

![snackbar](https://i.imgur.com/GJoEEjd.png)

**Comment by fkhadra** [comment]
Hey @goforbg , I'm a bit late. You can render anything, sky is the limit 😆. Have a look at this example https://fkhadra.github.io/react-toastify/add-an-undo-action-to-a-toast/

**Reply by goforbg** [reply]
That's cool! Exactly what I wanted thanks!
